src/Core/Services/MetadataProviders/MsSqlMetadataProvider.cs

```diff
@@ -236,5 +240,24 @@ private bool TryResolveDbType(string sqlDbTypeName, out DbType dbType)
                 return false;
             }
         }
+
+        /// <summary>
+        /// This method returns whether the given value is a built_in function or not.
+        /// built_in funtions are surrounded by single parenthesis,i.e. (getdate())
+        /// string will be with parenthesis and single quotes, i.e. ('value')
+        /// int/float will be with double parenthesis, i.e. ((20))
+        /// </summary>
+        /// <param name="columnDefinitionDefaultValue">given value for database column</param>
+        /// <returns>true if given value is a built-in function</returns>
+        private static bool IsBuiltInFunction(string columnDefinitionDefaultValue)
+        {
+            if ((columnDefinitionDefaultValue.StartsWith("('") && columnDefinitionDefaultValue.EndsWith("')"))
+                || (columnDefinitionDefaultValue.StartsWith("((") && columnDefinitionDefaultValue.EndsWith("))")))
```

Comment:
Is there a separate issue created for mysql/postgresql assuming this logic wont work for those db types?

Reply:
Actually looks like we have added tests for other db types as well. In thast case, shouldn't this function be in the base claas. And each derived db type metadataprovider inherit from that class?

Reply:
other Dbs are handling it differently than MSSQL. While MsSQL drivier is not parsing the inbuilt function, mysql and postgres are able to do it, due to which this issue is specific to MsSQL, which is verfied with the tests as well.

Reply:
In that case, wouldn't MySQL/Postgres insert wrong values for default if in_built function is get_current_date() for example. If their drivers parse it and provide the current date value when GraphQL schema is generated, it would insert incorrect value at request time. How do we handle such incorrect mutations?

Reply:
i may not have understood your point, but the problem with mssql is it is considering built_in function as string, so we let database handle it, but for other dbs, it's handling the built_in functions correctly so we insert exactly aligned with the column type.